pkg/util/util.go

```diff
@@ -21,14 +21,17 @@ import (
 	"strings"
 )
 
-// ToCanonicalName converts Golang package/type name into canonical OpenAPI name.
+// ToRESTFriendlyName converts Golang package/type canonical name into REST friendly OpenAPI name.
 // Examples:
 //	Input:  k8s.io/api/core/v1.Pod
 //	Output: io.k8s.api.core.v1.Pod
 //
 //	Input:  k8s.io/api/core/v1
 //	Output: io.k8s.api.core.v1
-func ToCanonicalName(name string) string {
+//
+//	Input:  csi.storage.k8s.io/v1alpha1.CSINodeInfo
+//	Output: io.k8s.storage.csi.v1alpha1.CSINodeInfo
+func ToRESTFriendlyName(name string) string {
```

Comment:
I probably wouldn't change the exported function

Reply:
marked the function with misleading name as deprecated